test/testhelpers.sh

```diff
@@ -428,7 +428,19 @@ comparison_to_operator() {
 }
 
 calc_oid() {
-  printf "$1" | shasum -a 256 | cut -f 1 -d " "
+  if [ $IS_WINDOWS -eq 1 ]; then
+    printf "$1" | sha256sum
+  else
+    printf "$1" | shasum -a 256
+  fi | cut -f 1 -d " "
+}
+
+calc_oid_file() {
```

Comment:
Sure, but could you be a bit more clear on what to document? I mean, should I basically just use the commit message and document *why* to use sha256sum on Windows, or do you want to have the pipe of commands itself documented?